parametric/conftest.py

```diff
@@ -161,7 +161,7 @@ def golang_library_factory(env: Dict[str, str]):
 def dotnet_library_factory(env: Dict[str, str]):
     dotnet_appdir = os.path.join("apps", "dotnet")
     dotnet_dir = os.path.join(os.path.dirname(__file__), dotnet_appdir)
-    dotnet_reldir = os.path.join("parametric", dotnet_appdir)
+    dotnet_reldir = os.path.join("parametric", dotnet_appdir).replace("\\", "/")
```

Comment:
how does "\\" end up being in the string 🤔

Reply:
Based on the docs, `os.path.join` will return a string with the OS-specific path separators. On Windows where `\` is a path separator, it returns a string with `\` instead of `/`, and I think this throwing off some of the dockerfile building. So I'm remapping them to `/` for the rest of the operations

Reply:
ah right, we're going back into a linux docker container with this string, makes sense!